File: Zadania/2025/Zadanie-4/Java/Alfabet.java
import java.util.ArrayList;
import java.util.List;

public class Alfabet {
    List<String> litery ;

    Alfabet(){
        litery =  new ArrayList<String>();   
    }
    public void dodaj(String litera){
        this.litery.add(litera);
    }
    public void usun(int pozycja){
        this.litery.remove(pozycja);
    }
    public void wypisz(){
        System.out.println(litery);
    }
    public static void main(String[] args) {
        Alfabet Alfabet = new Alfabet();
        Alfabet.dodaj("g");
        Alfabet.dodaj("c");
        Alfabet.dodaj("d");
        Alfabet.dodaj("f");
        Alfabet.dodaj("j");
        Alfabet.dodaj("x");
        Alfabet.dodaj("b");
        Alfabet.usun(2);
        Alfabet.usun(1);
        Alfabet.dodaj("a");
        Alfabet.dodaj("k");
        Alfabet.dodaj("m");
        Alfabet.wypisz();
    }
}

File: Zadania/2025/Zadanie-4/Python/Alfabet.py
class Alfabet:
    def __init__(self):
        self.litery = []

    def dodaj(self , litera):
        self.litery.append(litera)

    def usun(self , ktoraLitera):
        if 0 <= ktoraLitera < len(self.litery):  
            self.litery.pop(ktoraLitera)
        else:
            print(f"Nieprawidłowa pozycja: {ktoraLitera}")

    def wypisz(self):
        print (self.litery)

def alfabet():  
    alfabet = Alfabet()
    alfabet.dodaj("a")
    alfabet.dodaj("b")
    alfabet.dodaj("c")
    alfabet.dodaj("d")
    alfabet.dodaj("e")
    alfabet.dodaj("f")
    alfabet.dodaj("g")
    alfabet.dodaj("h")
    alfabet.dodaj("i")
    alfabet.dodaj("j")
    alfabet.dodaj("k")
    alfabet.usun(2)
    alfabet.usun(5)
    alfabet.usun(6)
    alfabet.wypisz()

alfabet()



